### DISCUSSION TITLE
Sometimes I cannot  enter the workspace of Theia

### DISCUSSION THREAD

**Discussion by zhuanshenlikai** [eclipse-theia/theia#13635] [Q&A]
After starting Theia, it keeps spinning and the background keeps creating sockets, but it cannot enter the workspace.I have created a Docker image of Theia and it is running in a container environment.
Here is a screenshot of the exception message:
The Theia version: 1.47.1

![1713943022421](https://github.com/eclipse-theia/theia/assets/12570095/f41a8697-9c75-4fb6-9db6-09784256745d)

**Comment by msujew** [comment]
Hey @zhuanshenlikai,

do you get any errors in the (browser) console? Does this also happen when running on your local system?
